### PR TITLE
✨ Clinical Submission pending approval state

### PR DESCRIPTION
**Pull request by hlminh2000**
**Description of changes**

Rendering the approval pending state of clinical submission

**Type of Change**

- [ ] Bug
- [x] Styling
- [ ] New Feature

**Checklist before requesting review:**

- [ ] Matches design:

  - component sizes, spacing, and styles
  - font size, weight, colour
  - spelling has been double checked

- [x] New uikit components have Storybook stories
- [ ] Feature is minimally responsive.
- [x] Manual testing of UI feature.
- [ ] Selenium test is completed and passing.

### CODE REVIEW

File: uikit/Progress/index.tsx [hunk 1]
```diff
@@ -85,38 +81,44 @@ const Separator = styled<'div', { state: ProgressStatus }>('div')`
...
+    [PROGRESS_STATUS.LOCKED]: <Icon width="10px" height="10px" fill="white" name="lock" />,
+    [PROGRESS_STATUS.DISABLED]: null,
+  }[state]);
 
 export const ProgressItem = ({
   state,
   text,
   completed = state === PROGRESS_STATUS.SUCCESS,
+  className = '',
+  ...rest
```
Comment (joneubank) [inline uikit/Progress/index.tsx]: this value really representing `props`?

Comment (hlminh2000) [reply uikit/Progress/index.tsx]: the type def for props includes `React.HTMLAttributes<HTMLDivElement>` so `rest` would be inferred to be `React.HTMLAttributes<HTMLDivElement>` minus `className` :)

Comment (joneubank) [reply uikit/Progress/index.tsx]: coolio

**Comment by hlminh2000** [comment]
@joneubank thanks! I'll submit a separate cleanup PR to organize the file better :)

**Pull request merged**